gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/ui/SnappyRegionStatsCollectorFunction.java

```diff
@@ -40,7 +40,11 @@
 
   public static String ID = "SnappyRegionStatsCollectorFunction";
 
-  public static final ObjectSizer sizer = ObjectSizer.getInstance(false);
+  public static final ObjectSizer sizer = ObjectSizer.getInstance(true);
+
+  static {
+    sizer.setForInternalUse(true);
```

Comment:
To create a new instance with isInternalUse flag set as true. The flag when set as true, doesn't log un-neccessary messages.